src/provisioner/config_store.py

```diff
@@ -30,9 +30,16 @@ def set_kvs(self, kvs: list):
         Parameters:
         kvs - List of KV tuple, e.g. [('k1','v1'),('k2','v2')]
         """
-
-        for key, val in kvs:
-            Conf.set(self._conf_idx, key, val)
+        for key, val in kvs.copy():
```

Comment:
Why copy and remove ? This looks unnecessary.

Reply:
Removed this code, as it is not required anymore.